{"name": "Login to the app and navigate to ranking", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id='orderArea']/div[2]/button[2]\"}\n  (Session info: chrome=133.0.6943.137); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n0   chromedriver                        0x000000010c482d98 chromedriver + 5954968\n1   chromedriver                        0x000000010c47a9ca chromedriver + 5921226\n2   chromedriver                        0x000000010bf3a5e0 chromedriver + 415200\n3   chromedriver                        0x000000010bf8be1d chromedriver + 749085\n4   chromedriver                        0x000000010bf8bfc1 chromedriver + 749505\n5   chromedriver                        0x000000010bfdc104 chromedriver + 1077508\n6   chromedriver                        0x000000010bfb203d chromedriver + 905277\n7   chromedriver                        0x000000010bfd939e chromedriver + 1065886\n8   chromedriver                        0x000000010bfb1de3 chromedriver + 904675\n9   chromedriver                        0x000000010bf7dc6a chromedriver + 691306\n10  chromedriver                        0x000000010bf7edc1 chromedriver + 695745\n11  chromedriver                        0x000000010c445e50 chromedriver + 5705296\n12  chromedriver                        0x000000010c449da0 chromedriver + 5721504\n13  chromedriver                        0x000000010c427797 chromedriver + 5580695\n14  chromedriver                        0x000000010c44a82b chromedriver + 5724203\n15  chromedriver                        0x000000010c415fc4 chromedriver + 5509060\n16  chromedriver                        0x000000010c469088 chromedriver + 5849224\n17  chromedriver                        0x000000010c46924f chromedriver + 5849679\n18  chromedriver                        0x000000010c47a5a8 chromedriver + 5920168\n19  libsystem_pthread.dylib             0x00007ff80f436253 _pthread_start + 99\n20  libsystem_pthread.dylib             0x00007ff80f431bef thread_start + 15\n\n", "trace": "  File \"/Users/choesuyang/Desktop/ProjectChallenge/pyenv/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/Users/choesuyang/Desktop/ProjectChallenge/pyenv/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features/steps/login.py\", line 59, in step_impl\n    context.product_page.click_the_option_button()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"/Users/choesuyang/Desktop/ProjectChallenge/features/pages/ProductPage.py\", line 21, in click_the_option_button\n    purchase_button = self.driver.find_element(By.XPATH, \"//*[@id='orderArea']/div[2]/button[2]\")\n  File \"/Users/choesuyang/Desktop/ProjectChallenge/pyenv/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/choesuyang/Desktop/ProjectChallenge/pyenv/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/Users/choesuyang/Desktop/ProjectChallenge/pyenv/lib/python3.13/site-packages/appium/webdriver/errorhandler.py\", line 125, in check_response\n    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))\n"}, "steps": [{"name": "Given click the option button", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//*[@id='orderArea']/div[2]/button[2]\"}\n  (Session info: chrome=133.0.6943.137); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n0   chromedriver                        0x000000010c482d98 chromedriver + 5954968\n1   chromedriver                        0x000000010c47a9ca chromedriver + 5921226\n2   chromedriver                        0x000000010bf3a5e0 chromedriver + 415200\n3   chromedriver                        0x000000010bf8be1d chromedriver + 749085\n4   chromedriver                        0x000000010bf8bfc1 chromedriver + 749505\n5   chromedriver                        0x000000010bfdc104 chromedriver + 1077508\n6   chromedriver                        0x000000010bfb203d chromedriver + 905277\n7   chromedriver                        0x000000010bfd939e chromedriver + 1065886\n8   chromedriver                        0x000000010bfb1de3 chromedriver + 904675\n9   chromedriver                        0x000000010bf7dc6a chromedriver + 691306\n10  chromedriver                        0x000000010bf7edc1 chromedriver + 695745\n11  chromedriver                        0x000000010c445e50 chromedriver + 5705296\n12  chromedriver                        0x000000010c449da0 chromedriver + 5721504\n13  chromedriver                        0x000000010c427797 chromedriver + 5580695\n14  chromedriver                        0x000000010c44a82b chromedriver + 5724203\n15  chromedriver                        0x000000010c415fc4 chromedriver + 5509060\n16  chromedriver                        0x000000010c469088 chromedriver + 5849224\n17  chromedriver                        0x000000010c46924f chromedriver + 5849679\n18  chromedriver                        0x000000010c47a5a8 chromedriver + 5920168\n19  libsystem_pthread.dylib             0x00007ff80f436253 _pthread_start + 99\n20  libsystem_pthread.dylib             0x00007ff80f431bef thread_start + 15\n\n", "trace": "  File \"/Users/choesuyang/Desktop/ProjectChallenge/pyenv/lib/python3.13/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^\n  File \"/Users/choesuyang/Desktop/ProjectChallenge/pyenv/lib/python3.13/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"features/steps/login.py\", line 59, in step_impl\n    context.product_page.click_the_option_button()\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"/Users/choesuyang/Desktop/ProjectChallenge/features/pages/ProductPage.py\", line 21, in click_the_option_button\n    purchase_button = self.driver.find_element(By.XPATH, \"//*[@id='orderArea']/div[2]/button[2]\")\n  File \"/Users/choesuyang/Desktop/ProjectChallenge/pyenv/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 770, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/choesuyang/Desktop/ProjectChallenge/pyenv/lib/python3.13/site-packages/selenium/webdriver/remote/webdriver.py\", line 384, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"/Users/choesuyang/Desktop/ProjectChallenge/pyenv/lib/python3.13/site-packages/appium/webdriver/errorhandler.py\", line 125, in check_response\n    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))\n"}, "start": 1743832936426, "stop": 1743832937130}], "start": 1743832936425, "stop": 1743832937131, "uuid": "64dcabd9-c80c-45b2-9334-64670aa6e096", "historyId": "c5e4be62ebf4e60e34a183b57b8d9356", "fullName": "Login and navigate through the application: Login to the app and navigate to ranking", "labels": [{"name": "owner", "value": "SuyangChoi"}, {"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login and navigate through the application"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}], "links": [{"type": "link", "url": "https://dev.example.com/", "name": "https://dev.example.com/"}, {"type": "issue", "url": "UI-123", "name": "UI-123"}]}